opencontext_py/tests/unit/searcher/new_solrsearcher/test_sorting.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import logging
+from opencontext_py.apps.searcher.new_solrsearcher import configs
+from opencontext_py.apps.searcher.new_solrsearcher.sorting import SortingOptions
+
+logger = logging.getLogger("tests-unit-logger")
+
+
+TESTS_REQUEST_TO_SOLR = [
+    # Tuples of test cases, with input dicts and expected output strings:
+    #
+    # (request_dict, sort value for solr),
+    #
+    ({}, configs.SOLR_SORT_DEFAULT,),
+    ({'sort':[]}, configs.SOLR_SORT_DEFAULT,),
+    ({'sort':False}, configs.SOLR_SORT_DEFAULT,),
+    
+    # Test cases on interest score sorting
+    ({'sort':'interest'}, 'interest_score asc, sort_score asc, slug_type_uri_label asc',),
```

Comment:
I'm surprised that `configs.SOLR_SORT_DEFAULT` is `interest_score desc` but that `{'sort':'interest'}` should map to `interest_score asc`.  Maybe I'll get the logic as I go along, but this is my first reaction.

Reply:
Yep, it's a little confusing, since unless specified as DESC, any sorting is assumed to be in ascending order. So, if the client requests sorting on interest without saying DESC they get this not so useful sort order.